src/components/Comments.jsx

```diff
@@ -35,7 +41,60 @@ function Comments({ articleId }) {
       </Paper>
     );
 
-  return <>{isLoading ? <p>Loading...</p> : commentElements}</>;
+  const queueSnackbar = (message, variant) => {
+    enqueueSnackbar(message, {
+      variant: variant,
+      anchorOrigin: { vertical: "top", horizontal: "center" },
+      action: (key) => (
+        <Button onClick={() => closeSnackbar(key)}>Dismiss</Button>
+      ),
+    });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const comment = e.target.comment.value;
+    const username = e.target.username.value;
```

Comment:
Users should not be allowed to post as anybody else. We should have a user system. It can be hard coded 🙂